web/gui/dashboard/src/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import { Provider } from "react-redux"
+
+import { configureStore } from "store"
+
+import "./index.css"
+import App from "./App"
+import * as serviceWorker from "./serviceWorker"
+
+const store = configureStore()
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById("root"),
+)
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister()
```

Comment:
Is the CRA service worker needed for some purpose? Otherwise we can safely remove it.